pyproject.toml

```
@@ -29,6 +29,7 @@ documentation = "https://stfc.github.io/aiida-mlip/"
 python = "^3.9"
 aiida-core = "^2.5"
 voluptuous = "^0.14"
+PyCifRW = "^4.4.6"
```

```suggestion
ase = "^3.22.1"
PyCifRW = "^4.4.6"
voluptuous = "^0.14"
```
I think ASE needs to be added as dependency somewhere to stop the pre-commit failing.

I possibly wouldn't actually commit this suggestion currently, since it's only actually used for the example, but it'd be needed if you use ASE in the main code.

---

Oh not sure if it was before, but it's needed in `aiida_mlip/parsers.py`, so you probably do need to add it as a dependency?